css/Components/Root.scss

```diff
@@ -0,0 +1,5 @@
+@import "../Theme/Color";
+
+body {
+  --bg-color: $Color-deep-cove;
```

Comment:
I don't agree with naming it after a Suit component. that assumes you're following Suit rules for this, which is not the case. You'd end up using a `Layout-bg-color` variable inside the Hero component